color-picker/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+import org.jetbrains.compose.compose
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose") version "1.0.0-beta5"
+    id("com.android.library")
+    id("kotlin-android-extensions")
+    id("com.vanniktech.maven.publish")
+}
+
+group = "com.godaddy"
+version = "1.0"
+
+kotlin {
+    android()
+    jvm("desktop") {
+        compilations.all {
+            kotlinOptions.jvmTarget = "11"
+        }
+    }
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                api(compose.runtime)
+                api(compose.foundation)
+                api(compose.material)
+                implementation("com.github.ajalt.colormath:colormath:3.1.1")
```

Comment:
Mostly thinking out loud here, but I'm wondering if there isn't a way to only use this dependency only if its the Desktop version, and if that makes sense to do. I think all we need it for is color conversions to and from HSV right?

Reply:
It's true that we can rely on this only on the desktop, but there are a lot of color conversion tools in this library, and since I don't know so much about color things , it might be useful to keep it if you want to expand other functions for this library in the future,  What do you think?

Reply:
Let me give it a bit more thinking and chat with the rest of the GoDaddy Studio Android Team about it tomorrow. 

Otherwise, the PR looks good and it is now working for me on Desktop. I'll revert back tomorrow 👍🏻

Reply:
Alright I think we are good to go with this library 💯  Thanks so much for the contribution, merging it now.